src/main/java/pl/wkos/homework24/service/ProductService.java

```diff
@@ -0,0 +1,13 @@
+package pl.wkos.homework24.service;
+
+import pl.wkos.homework24.model.Product;
+
+import java.util.Set;
+
+public class ProductService {
+    public static double getPriceOfAllProducts(Set<Product> products){
```

Comment:
Ta metoda nie powinna być statyczna. Zamiast tego powinna być normalną metodą serwisu. Jak chcesz tworzyć obiekt pośredni w postaci serwisu to najlepiej jakbyś tutaj sobie (do serwisu) wstrzyknął repozytorium, odwzorował w nim metody typu `findAll`, `getPriceOfAllProducts` i korzystał z tego serwisu w kontrolerze. Wtedy zrobisz to po bożemu - obiekt serwisu będzie pośredniczył w komunikacji pomiędzy repozytorium a kontrolerem, gdzie można zawrzeć dodatkową logikę.

Reply:
Tak chciałem zrobić, ale czasu brakło. Poprawię.

Reply:
Poprawiłem. Możesz sprawdzić?